.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--
```

Comment:
I appreciate that the suggestions in these templates are commented out, so they won't end up in every issue...